autogpt_platform/frontend/src/app/[lang]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import { getDictionary } from "./dictionaries";
+
+export default async function Page({
+  params: { lang },
+}: {
+  params: { lang: string };
+}) {
+  const dict = await getDictionary(lang); // en
+  return <h1>{dict.home.welcome}</h1>; // Add to Cart
```

Comment:
??? Add to cart?

Reply:
Just copy and paste example page for using lang.

This will be changed before merge. Thanks for pointing it out